gallery/tutorials/cov3d_simulation.py

```diff
@@ -31,15 +31,26 @@
 img_size = 8  # image size in square
 num_imgs = 1024  # number of images
 num_eigs = 16  # number of eigen-vectors to keep
+dtype = np.float32
+
+# Generate a ``Volume`` object for use in the simulation.
```

Comment:
Say something like, "here we use an older `LegacyVolume` for simplicity. Other options include `AsymmetricVolume`, ... [other volumes]"?

Reply:
I'm hesitant to make this suggestion here because the verification tools used below in this tutorial have some bugs (we're working on) that will be exposed by changing the volume.

Reply:
Sounds good, disregard